src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -78,16 +77,14 @@ public class ExportController : Controller
         [Route(KnownRoutes.Export)]
         [ServiceFilter(typeof(ValidateExportRequestFilterAttribute))]
         [AuditEventType(AuditEventSubType.Export)]
-        public async Task<IActionResult> Export(
-            [FromQuery(Name = KnownQueryParameterNames.DestinationType)] string destinationType,
-            [FromQuery(Name = KnownQueryParameterNames.DestinationConnectionSettings)] string destinationConnectionString)
+        public async Task<IActionResult> Export()
         {
             if (!_exportConfig.Enabled)
             {
                 throw new RequestNotValidException(string.Format(Resources.UnsupportedOperation, OperationsConstants.Export));
             }
 
-            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, destinationType, destinationConnectionString, HttpContext.RequestAborted);
+            CreateExportResponse response = await _mediator.ExportAsync(_fhirRequestContextAccessor.FhirRequestContext.Uri, HttpContext.RequestAborted);
```

Comment:
A few more things we should address (doesn't necessarily need to be in this PR).

1. If the user cancels the export or if export fails, we should delete all exported data from the storage.
2. We need to figure out how to expire exported results for completed job.

Reply:
TTL on the document could be a solution for this (but would need a different solution for sql)

Reply:
Created work items to track these issues (and other nice to haves)